docs/usage.md

```diff
@@ -82,78 +134,80 @@ By default, the pipeline expects paired-end data. If you have single-end data, y
 
 It is not possible to run a mixture of single-end and paired-end files in one run.
 
+## Optional arguments
 
-## Reference Genomes
+### Trimming options:
 
-The pipeline config files come bundled with paths to the illumina iGenomes reference index files. If running with docker or AWS, the configuration is set up to use the [AWS-iGenomes](https://ewels.github.io/AWS-iGenomes/) resource.
+    --adapter_forward             Sequence of 3' adapter to remove in the forward reads
```

Comment:
Hah, nice try! I think it's better to properly put these in (can still be fairly quick copy and paste with multiple cursors in Sublime / Atom etc). With headings it's possible to directly link to the docs for an option which is super helpful, and quite a few of these deserve a little more explanation than their one-liners.. Also, it's good to be consistent with the other options in this readme.